{
  "loc.friendlyName": "Autenticación de la carga de Python con Twine",
  "loc.helpMarkDown": "[Obtener más información acerca de esta tarea](https://docs.microsoft.com/azure/devops/pipelines/tasks/package/twine-authenticate) o [consultar la documentación de Twine](https://twine.readthedocs.io/)",
  "loc.description": "Realice la autenticación para cargar distribuciones de Python con Twine. Agregue \"-r FeedName/EndpointName --config-file $(PYPIRC_PATH)\" al comando de carga de Twine. Para las fuentes presentes en la organización, use el nombre de la fuente como repositorio (-r). De lo contrario, use el nombre del punto de conexión definido en la conexión de servicio.",
  "loc.instanceNameFormat": "Autenticación de Twine $(message)",
  "loc.group.displayName.feedAuthentication": "Fuentes y autenticación",
  "loc.group.displayName.advanced": "Avanzado",
  "loc.input.label.artifactFeed": "Mi nombre de fuente (selecciónelo a continuación)",
  "loc.input.help.artifactFeed": "Seleccione una fuente para autenticar que esté presente en la organización.",
  "loc.input.label.pythonUploadServiceConnection": "Fuente de organizaciones externas",
  "loc.input.help.pythonUploadServiceConnection": "Seleccione un punto de conexión para autenticarse fuera de la organización y asegúrese de que este tiene permisos de carga de paquetes.",
  "loc.input.label.workloadIdentityServiceConnection": "Conexión de servicio \"Usuario de Azure DevOps respaldado por Id. de carga de trabajo de Entra\"",
  "loc.input.help.workloadIdentityServiceConnection": "Si se establece, se requiere feedUrl. Se omiten todas las demás entradas.",
  "loc.input.label.feedUrl": "Dirección URL de la fuente de Azure Artifacts.",
  "loc.input.help.feedUrl": "Si se establece, se requiere workloadIdentityServiceConnection. Se omiten todas las demás entradas. No es compatible con pythonUploadServiceConnections. La dirección URL de la fuente debe tener el formato del Registro de carga pypi, por ejemplo, https://pkgs.dev.azure.com/{ORG_NAME}/{PROJECT}/_packaging/{FEED_NAME}/pypi/upload/",
  "loc.messages.Info_AddingInternalFeeds": "Agregando información de autenticación para la fuente interna %s.",
  "loc.messages.Info_AddingExternalFeeds": "Agregando información de autenticación para el punto de conexión externo %s.",
  "loc.messages.Info_AddingAuthForRegistry": "Agregando autenticación a la configuración del registro %s",
  "loc.messages.Info_SuccessAddingAuth": "La autenticación para la fuente interna %s y el punto de conexión externo %s se ha agregado correctamente.",
  "loc.messages.Info_AddingPasswordAuthEntry": "Agregando la entrada de autenticación de contraseña del nombre de usuario para la fuente %s.",
  "loc.messages.Info_AddingTokenAuthEntry": "Agregando la entrada de autenticación de token para la fuente %s.",
  "loc.messages.Info_SuccessAddingFederatedFeedAuth": "La autenticación federada para la fuente %s se agregó correctamente.",
  "loc.messages.Error_FailedToParseFeedUrlAndAuth": "No se pudo analizar la dirección URL de la fuente ni agregar la información de autenticación. %s",
  "loc.messages.Warning_OnlyOneFeedAllowed": "Solo se puede autorizar una fuente interna a la vez con esta tarea. Se ignorarán todas las fuentes excepto la primera.",
  "loc.messages.Warning_OnlyOneServiceConnectionAllowed": "Solo se puede autorizar una conexión de servicio a la vez con esta tarea. Se ignorarán todas las conexiones excepto la primera.",
  "loc.messages.Error_DuplicateEntryForFeed": "This is a duplicate entry for the feed name %s. Use unique names to avoid potential publish issues.",
  "loc.messages.FailedToGetPackagingUri": "No se puede obtener el URI de empaquetado; se usa el URI de la colección predeterminado.",
  "loc.messages.FailedToAddAuthentication": "No se pudo agregar la autenticación",
  "loc.messages.NoPypircFile": "No hay ningún archivo .pypirc para limpiar.",
  "loc.messages.VariableSetForPypirc": "La variable de entorno para .pypirc en %s está establecida.",
  "loc.messages.Info_RemovingPypircFile": "Quitando el archivo .pypirc de %s",
  "loc.messages.NoRepoFound": "No se encontró ninguna dirección URL de repositorio en %s",
  "loc.messages.Error_FailedToGetServiceConnectionAuth": "No se pueden obtener las credenciales federadas de la conexión de servicio: %s.",
  "loc.messages.Error_MissingFeedUrlOrServiceConnection": "Tanto la dirección URL de la fuente como la conexión de servicio deben establecerse y no pueden estar vacías."
}